feature-utils/poly-look/src/react-components/inputs/common.css

```diff
@@ -0,0 +1,90 @@
+.poly-input-field {
+  display: flex;
+  flex-direction: column;
+}
+
+.poly-input-field .body {
+  height: 56px;
+  border: 1px solid var(--pl-field-color-border-enabled);
+  padding: var(--pl-field-spacing-padding-vert)
+    var(--pl-field-spacing-padding-hor);
+  border-radius: var(--pl-radius-1x);
+  display: flex;
+  align-items: center;
+  position: relative;
+  box-sizing: border-box;
+  background-color: var(--pl-field-color-fill-enabled);
+  font-size: var(--font-size-ml);
+  line-height: var(--pl-field-body-line-height);
+}
+
+.poly-input-field .body-focused {
+  background-color: var(--pl-field-color-fill-focused);
+  border-color: var(--pl-field-color-border-focused);
+}
+
+.poly-input-field .body-error {
+  background-color: var(--pl-field-color-fill-error);
+  border-color: var(--pl-field-color-border-error);
+}
+
+.poly-input-field .body-error + .helper-text {
+  color: var(--pl-field-color-text-error);
+}
+
+.poly-input-field .body-disabled {
+  pointer-events: none;
+  background-color: var(--pl-field-color-fill-disabled);
+  border-color: var(--pl-field-color-border-disabled);
+  color: var(--pl-field-color-text-disabled);
+}
+
+.poly-input-field .body-disabled + .helper-text {
+  color: var(--pl-field-color-text-disabled);
+}
+
+.poly-input-field input {
+  flex: 1;
+  outline: none;
+  border: none;
+  padding: 0;
+  font: inherit;
+  line-height: inherit;
+  letter-spacing: inherit;
+  background-color: inherit;
+  margin-right: var(--pl-field-spacing-gap);
+}
+
+.poly-input-field label {
+  position: absolute;
+  font: inherit;
+  line-height: inherit;
+  letter-spacing: inherit;
+  left: var(--pl-field-spacing-padding-hor);
+  top: 50%;
+  transform: translateY(-50%);
+  transition: transform 0.2s, font-size 0.2s;
+}
+
+.poly-input-field .label-up {
+  font-size: var(--font-size-s);
+  transform: translateY(calc(-1 * var(--pl-field-body-line-height)));
+}
+
+.poly-input-field .label-up + input {
+  margin-bottom: calc(-1 * calc(56px / 4));
```

Comment:
One thing to simplify this would be to use `align-self: flex-end;` so that the input sits on the bottom padding of the input.
Also you can probably already add this without the class .label-up since the input is hidden initally

Reply:
Done. Thank you! :D